@ecomplus/widget-product/src/components/js/EcProduct.js

```diff
@@ -73,6 +73,14 @@ export default {
       return this.selectedVariation.name || name(this.body)
     },
 
+    selectedVariationQuantity () {
```

Comment:
Acho que o nome deste computed não faz sentido pro que você tá colocando nele, embora você tenha chamado de `selectedVariationQuantity`, ele será 0 quando a variação selecionada tiver quantidade menor que 13 (este número mockado de novo haha).

Ficou confuso quando eu vi no template, se você chamou de `selectedVariationQuantity` ele deveria ser sempre a "quantidade da vairação seleciona", faz sentido? E não é isto que acontece...

Reply:
Basicamente isso é um gatilho para quando tem menos de 13 em estoque. O número 13 é apenas um número pelos sites que olhei, poderia ser 11 por exemplo. Pois navegando pelo site da Amaro, já vi variações com 12 produtos em estoque mostrando a mensagem, faltam apenas 12 itens em estoque. Nunca acima disso, vi em outros sites também que ficam na margem de 10. Como o maior que vi era 12, então escolhi o parâmetro 13 para que quando for abaixo disso, eu mostrar esse gatilho para que o cliente final não perca oportunidade. Se não tiver nessa faixa eu nem considero, por isso retorno 0. Mas posso tirar isso, deixar retornar o valor certo da variação para um valor diferente disso, só achei que seria mais limpo dessa forma

Reply:
Eu entendi depois, o problema é o nome que você escolheu pro computed (naming things é complicado mesmo), entendeu?
Você pode manter sua implementação, só acho que pra manter desta forma deveria escolher um nome que fosse mais descritivo pra este computed.

Sobre o 13, eu imaginei que você tinha pegado algumas referências, mas mesmo assim acho que poderia ser uma prop pelo menos, é uma implementação bem simples e desta forma permitimos que o valor seja customizado, sacou?

Reply:
Beleza, saquei, coloquei uma prop. E alterei lá com um computed pra mostrar só a quantidade seja variação ou não. E outro pra mostrar quando tiver a quantidade minima de estoque

Reply:
Ficou bala!